releases/22.10.0/22100 feature list.md

```diff
@@ -34,6 +34,7 @@ The general theme of this release is "TBD".
 
 
 **sig-network**
+* Multiple improvements to the client server connection experience, including debug text to convery current step in connection process, along with connection status ImGui display and new ImGUI menu options.
```

Comment:
“Multiple improvements to the client and server connection experience, including debug text in the viewport to convery current step in connection process, connection status, and new ImGUI menu options.”
- Changed “client server” to “client and server” just to be sure it’s not confused with the client server host type.
- Connection status is displayed as debug text in the viewport separate from the ImGui menu (although it was suggested adding it there as well)

Reply:
See new edit for update